food-truck-frontend/src/components/recommended-trucks/SearchRecommendedTrucksForm.tsx

```diff
@@ -64,6 +65,15 @@ class RecommendedTrucksForm extends Component<
     );
   }
 
+  getRouteLocations(trucks: RecommendedSimpleTruck[]) {
+      var routeLoc: any = [];
+      trucks.map((t, ndx) => {
```

Comment:
I'd change this to `return trucks.map((t, ndx) => backendToFrontend(t.loc, ndx))`, personally, or use `forEach` instead of map (since otherwise we're throwing away the results of the map).